src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String done, type, description;
+
+    public Task(String description) {
+        this.done = " ";
+        this.type = " ";
+        this.description = description;
+    }
+
+    public void print(){}
```

Comment:
If the function was meant to be overridden, consider using an abstract function instead.